kubernetes/local-ingress.yml

```diff
@@ -9,36 +9,6 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
 spec:
   rules:
-    - host: executor.astria.localdev.me
-      http:
-        paths:
-          - path: "/"
-            pathType: Prefix
-            backend:
-              service:
-                name: evm-service
-                port:
-                  name: json-rpc-svc
-    - host: ws-executor.astria.localdev.me
-      http:
-        paths:
-          - path: "/"
-            pathType: Prefix
-            backend:
-              service:
-                name: evm-service
-                port:
-                  name: ws-rpc-svc
-    - host: faucet.astria.localdev.me
-      http:
-        paths:
-          - path: "/"
-            pathType: Prefix
-            backend:
-              service:
-                name: faucet-service
-                port:
-                  name: faucet-svc
     - host: celestia-token.astria.localdev.me
```

Comment:
Should we just remove this file entirely, ingress to grab the celestia token seems like more of a testing piece?